scripts/smeshr/prepare_HRRR_SC_inputs.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
```

Comment:
Style feedback - A environment independent definition would look like this:
```bash
!#/usr/bin/env bash
```
This doesn't assume that bash is installed under `/bin/bash` and lets the OS figure it out. Very minor though and your definition is usually supported. [Some curious reading](https://linuxize.com/post/bash-shebang/)

Reply:
Got it, thanks Joe!